scripts/ci/tags.yaml

```diff
@@ -477,3 +477,14 @@ cellular:
     - modules/lib/cjson/
     - modules/lib/hostap/
     - modules/lib/zcbor/
+
+fast_pair:
```

Comment:
Shouldn't we add `fast_pair` tag also to the Fast Pair samples? `nrf/samples/bluetooth/fast_pair`

Reply:
Those nrf/samples/bluetooth/fast_pair already have bluetooth tag.
So which one is more important? bluetooth or fast_pair for this sample.

Beside that samples and tests can have different decencies (usually samples are more wide, tests should be isolated).
For this particular case, I propose to keep it as it is.

Reply:
Well, samples can have multiple tags as far as I understand. The Fast Pair category is subcategory of Bluetooth

Sort of like Bluetooth Mesh

We have 3 top-level categories for Bluetooth samples in our documentation right now:

https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples.html

Bluetooth samples
Bluetooth Fast Pair samples
Bluetooth Mesh samples

Reply:
Ok, so to be more consistent, I can add bluetooth tag instead and update its decencies?
Indeed, there could be multiple tags for sample but for test_plan tagging mechanisms, I want to add decencies only for single one (to prevent unintentional removal of some tests from scope)

Reply:
I see that Mesh samples do not have a distinct tag associated with them - they use the `bluetooth` tag (also associated tests use the `bluetooth` tag).

Perhaps, we should use the `bluetooth` tag for the Fast Pair tests so that they have the same tags as Fast Pair samples.

Reply:
I see that bluetooth mesh has bluetooth tag

Reply:
> I see that Mesh samples do not have a distinct tag associated with them - they use the `bluetooth` tag (also associated tests use the `bluetooth` tag).
> 
> Perhaps, we should use the `bluetooth` tag for the Fast Pair tests so that they have the same tags as Fast Pair samples.

I will then adjust my PR :)

Reply:
thanks